From 380737d0bf4021434db4d5e479f0b8a2aece6ec9 Mon Sep 17 00:00:00 2001
From: Pavel Moravec <pmoravec@redhat.com>
Date: Thu, 4 Apr 2019 10:43:24 +0200
Subject: [PATCH] [pcp] collect pmlogger without a sizelimit

sizelimit=None defaults to --log-size, use sizelimit=0 instead

Resolves: #1632

Signed-off-by: Pavel Moravec <pmoravec@redhat.com>
---
 sos/plugins/pcp.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sos/plugins/pcp.py b/sos/plugins/pcp.py
index da7158a6..da2a6611 100644
--- a/sos/plugins/pcp.py
+++ b/sos/plugins/pcp.py
@@ -130,7 +130,7 @@ class Pcp(Plugin, RedHatPlugin, DebianPlugin):
             pmlogger_ls = self.get_cmd_output_now("ls -t1 %s" % path)
             if pmlogger_ls:
                 for line in open(pmlogger_ls).read().splitlines():
-                    self.add_copy_spec(line, sizelimit=None)
+                    self.add_copy_spec(line, sizelimit=0)
                     files_collected = files_collected + 1
                     if self.countlimit and files_collected == self.countlimit:
                         break
-- 
2.17.2

